ext/mysqli/mysqli.stub.php

```diff
@@ -1,13 +1,81 @@
 <?php
 
-/** @generate-function-entries */
+/** @generate-class-entries */
 
 final class mysqli_driver
 {
+    /** @var string|null */
+    public $client_info;
```

Comment:
Should I add the `@readonly` annotation to read-only properties?

Reply:
Another question is that probably all properties are declared with the `null` default value, however read handlers don't return this value. E.g. `mysqli_driver::$client_info` returns only `string`. Does this mean that I should disregard the default value? Can we also make the readonly properties typed?

Reply:
Mysqli is a tricky case (see also #6587). I think what you do here is fine as a starting point. In a followup, I think it would be fine to convert these into types properties without a default (as long as the type is guaranteed by the read_property handler).

Reply:
The plan sounds good to me! I'll provide the followup PR in the closer future :)